networks/11111/upgrade.json

```diff
@@ -0,0 +1,10 @@
+{
+    "precompileUpgrades": [
+        {
+            "feeManagerConfig": {
+                "adminAddresses": ["0x6f0f6DA1852857d7789f68a28bba866671f3880D"],
+                "blockTimestamp": "1660658400"
```

Comment:
```suggestion
                "blockTimestamp": 1660658400
```

Reply:
I think the stringified version was used in the upgrade.

Reply:
@darioush is right, this was a mistake in the original confluence doc. Just updated this in both confluence and the gist was already updated.